### PR TITLE
removed friendly_id, set up homepage platoon dropdown, made table scr…

### PR DESCRIPTION
**Pull request by Mic-J-Lee**
…olling with fixed names

### CODE REVIEW

File: app/controllers/assignments_controller.rb [hunk 1]
```diff
@@ -40,7 +40,9 @@ def payload
         cohort = Student.where(platoon: platoon)
         cohort.each do |cohort_member|
           if cohort_member.first_name
-            if pr_title_without_puller.include?(cohort_member.first_name.downcase) || branch_name_without_puller.include?(cohort_member.first_name.downcase)
+            pr_pair? = pr_title_without_puller.include?(cohort_member.first_name.downcase)
```
Comment (JYoung217) [inline app/controllers/assignments_controller.rb:43]: these two should be methods

File: app/controllers/home_controller.rb [hunk 2]
```diff
@@ -20,6 +23,10 @@ def index
         end 
       }
     end
+
+    #list of platoons for dropdown
+    @platoons = Student.pluck(:platoon).uniq - [nil]
```
Comment (JYoung217) [inline app/controllers/home_controller.rb:28]: this is very hacky. can you figure out how to remove `nil`s without this?

File: app/views/home/_student.html.erb [hunk 3]
```diff
@@ -1,8 +1,8 @@
   <%assignment_names.each_with_index do |assignment_name, index|%>
     <%if student_progress[student.github_handle][index] == 'complete'%>
-      <td title=<%=student.first_name%> style="padding: 8px 18px 3px 3px; background-color:lime; border: 1px solid gray">complete</td>
+      <td title=<%=student.first_name%> style="padding: 8px 18px 3px 3px; background-color:lime; ">complete</td>
```
Comment (JYoung217) [inline app/views/home/_student.html.erb:3]: these inline CSS things aren't best practice. i'd go back and get this hooked up to an actual stylesheet with proper `ids` and stuff

File: app/views/home/_table.html.erb [hunk 4]
```diff
@@ -0,0 +1,19 @@
+<table>
+  <tr>
+    <th></th>
+    <th>%</th>
+    <%assignment_names.each do |repo_name|%>
```
Comment (JYoung217) [inline app/views/home/_table.html.erb:5]: you don't need to pass it in as `locals` - the instance variable can be passed from controller to view

File: app/views/home/_table.html.erb [hunk 5]
```diff
@@ -0,0 +1,19 @@
...
+      <td style="padding: 10px; font-weight: bold"><%=repo_name%></td>
+    <%end%>
+  </tr>
+  <%students.each do |student|%>
+    <%completion_array = student_progress[student.github_handle]%>
```
Comment (JYoung217) [inline app/views/home/_table.html.erb:10]: views should just be views - the heavy lifting of logic should be on the controller. i'd do all the logic there and pass all of the things you need in your views as instance variables

**Comment by JYoung217** [comment]
im assuming that all of these code changes will actually work. you definitely need to clean up some of the code for longevity purposes. go ahead and change things based on my recommendations.

File: app/views/home/_student.html.erb (hunk 3 above)
Comment (Mic-J-Lee) [reply app/views/home/_student.html.erb:3]: I cant get the background-color to work!! I moved the rest out

**Pull request merged**